tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
OK, makes sense.  Can you add a comment explaining this in the relevant section of `zproject/settings.py`?  

I'm thinking something like:
```
# We need to have pgroonga before pg_catalog in the postgres search path, because pgroonga overrides the @@ operator from pg_catalog, and pg_catalog is searched first if not specified in the search path.
```

Also, you'll need to put this in `scripts/setup/postgres-init-db` as well (the version that runs for setting up production instances).